src/cast_core/src/CollectionScanner.cpp

```diff
@@ -417,6 +417,8 @@ void CollectionScanner::run() {
     std::optional<bsoncxx::types::b_timestamp> readClusterTime;
     for (auto&& config : _loop) {
         for (const auto&& _ : config) {
+            BOOST_LOG_TRIVIAL(debug)
```

Comment:
I might be overlooking something, but won't this log "actor started...actor finished" every time the actor executes an iteration inside the same phase? E.g. if InsertRemove runs a thousand times inside a single phase, won't that be a thousand lines of starting/stopping?

It seems like, when an actor starts acting in a phase for the first time it should print "starting", when the actor finishes for the final time it prints "stopping", and if the actor is noping it should print "nop"?

Maybe that can be achieved by putting "starting" inside the inner (iteration) loop, and put "finished" after the inner loop but still inside the outer (phase) loop? Not sure about nop cases.